pipeline/scripts/inference_pipeline.jl

```diff
@@ -0,0 +1,41 @@
+using DrWatson
+@quickactivate "Analysis pipeline"
+
+# include AnlysisPipeline module
+include(srcdir("AnalysisPipeline.jl"))
+
+@info("""
+      Running inference on truth data.
+
+      ---------------------------------------------
+      Currently active project is: $(projectname())
+      Path of active project: $(projectdir())
+      """)
+
+## Set up the truth Rt and save a plot of it
+using .AnalysisPipeline, Plots, JLD2, EpiAware, Distributions
+
+wkly_ar = AR(
+    damp_priors = [truncated(Normal(0.8, 0.05), 0, 1)],
```

Comment:
Is this from the method spec? It seems like its quite high - usually these tend to assume fairly small amounts of autoregressive behaviour)

Reply:
Good spot. This is a C&P from the getting started notebook.

Maybe indicates that we need a chat about priors... obviously random walk is AR(1) with the damp variable fixed at 1, so I think of this as a prior on the scale of "White noise" <-> "random walk". This is IMO the justification for truncating the distribution onto [0,1]. Having said that, it would seem to be more natural in that case to use a `Beta` distribution.

Reply:
I've gone with a Jeffrey Prior for the `[0, 1]` valued processes e.g `Beta(0.5, 0.5)`.